docs/installation/java-library.md

```diff
@@ -0,0 +1,383 @@
+
+### AMPL
+
+**powsybl-ampl-converter** *(Advanced user)*  
+This module provides an implementation of the converter API to allow a user to export a network to a set of tabular text
+files. As the format is really simple to parse, it's commonly used to integrate Powsybl with an optimal power flow using
+an [AMPL](../iidm/exporter/ampl.md) model or with Matlab.
+
+## Network additional features
+
+The artifacts described below provide additional features around the network.
+
+**powsybl-iidm-comparator** *(User)*  
+This module provides functions to help to compare IIDM networks.
+
+**powsybl-iidm-mergingview** *(Advanced user)*  
+This module provides an alternative way to merge several networks, keeping the underlying networks unchanged.
+
+**powsybl-iidm-reducer** *(Advanced user)*  
+This module provides classes to extract a sub area of a network.
+
+**powsybl-iidm-util** *(Advanced user)*  
+This module provides additional features around the network.
+
+**powsybl-iidm-test** *(Advanced user)*  
+This module provides factories to create simple networks.
+
+# Simulators
+
+## Load flow
+
+**powsybl-loadflow-api** *(User)*  
+This module provides a common API for power flow computation. You have to implement this API to use your power flow
+simulator through powsybl.
+
+**powsybl-open-loadflow** *(User)*  
+This module provides an implementation of the load flow API.
+
+**powsybl-loadflow-results-completion** *(Advanced user)*  
+This module implements power flow equations to initialize the P, Q, V and $$\theta$$ from the characteristics of the
+equipments. This module can be used as a [post processor](../iidm/importer/post-processor/LoadFlowResultsCompletionPostProcessor.md)
+of the import or with the [load flow validation](../loadflow/validation.md).
+
+**powsybl-loadflow-validation** *(Advanced user)*  
+This module provides functions to check the consistency of a power flow result. It can be use as a java library, but the
+easiest way to use it, it's through the [iTools CLI](../tools/loadflow-validation.md).
+
+## Contingencies
+
+The artifacts described below are used to model contingencies. Contingencies can then be used to run security analyses, with
+or without remedial actions.
+
+**powsybl-contingency-api** *(User)*  
+This module provides an API to model and create contingencies.
+
+**powsybl-contingency-dsl** *(User)*  
+This module provides classes to load [a contingency list from a groovy script](../contingencies/GroovyDslContingenciesProvider.md). This is a powerful way to create contingencies
+iterating over the equipments of network.
+
+## Security analyses
+
+**powsybl-security-analysis-api** *(User)*  
+This module provides a common API for security analyses computation. You have to implement this API to use your simulator
+through powsybl. We also provide an simple implementation that relies on a power flow simulator.  
```

Comment:
I think I don't understand which implementation you are talking about.

Reply:
I'm talking about the SecurityAnalysisImpl (aka [slow implementation](http://www.powsybl.org/docs/tools/security-analysis.html#slow-implementation)) described in the website.

Do you want I change a little bit the documentation to explain a little bit more?

Reply:
Yes it could be useful. And I think that we have to modify the page http://www.powsybl.org/docs/loadflow/security-analysis.html in order to mention that we have now an implementation of the LF API now and an other distributed by RTE.